src/Chapter1.hs

```diff
@@ -636,7 +658,12 @@ You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
 
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Int -> Int
+firstDigit n = findFirst n
+    where 
+      findFirst :: Int -> Int
+      findFirst x = 
+          if div (abs (x)) 10 == 0 then abs (x) else findFirst (div (abs (x)) 10)
```

Comment:
Nice! 💯 

Here, you see that some values are repeated in several places. That means, that you could move it into separate variable and reuse. You can use `where`, fo example 🙂